core/bentley/src/test/OneAtATime.test.ts

```diff
@@ -3,20 +3,21 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-import * as chai from "chai";
-import * as chaiAsPromised from "chai-as-promised";
+// import * as chai from "chai";
+// import * as chaiAsPromised from "chai-as-promised";
+import { assert, describe, expect, it } from "vitest";
 import { AbandonedError, OneAtATimeAction } from "../OneAtATimeAction";
 import { BeDuration } from "../Time";
 
 /* eslint-disable @typescript-eslint/no-floating-promises */
 
-const assert = chai.assert;
-const expect = chai.expect;
+// const assert = chai.assert;
+// const expect = chai.expect;
 
-chai.use(chaiAsPromised);
+// chai.use(chaiAsPromised);
 describe("OneAtATime test", () => {
 
-  it("OneAtATime", async () => {
+  it.skip("OneAtATime", async () => {
```

Comment:
idk if i ever finished refactoring this test

Reply:
This is a tricky one I skipped for now as well, I need to put this in the PR description

Reply:
make a todo in the pr description, and create issues so these things dont get lost

Reply:
Issue opened: https://github.com/iTwin/itwinjs-core/issues/7200